Extension/src/common.ts

```diff
@@ -910,6 +910,17 @@ export async function promptReloadWindow(message: string): Promise<void> {
     }
 }
 
+export async function promptCDDFailed(copy?: boolean): Promise<void> {
+    const copy_message: string = "Failed to copy declaration or definition to clipboard.";
+    const create_message: string = "Failed to create declaration or definition.";
```

Comment:
"Failed to create declaration or definition." error is determined by the native side logic. And native side sends a request to typescript to show the error message in a prompt.